tests/Integration/MockPimApiTrait.php

```diff
@@ -39,6 +39,10 @@ protected function mockDefaultPimAPIResponses(): void
             'get-product-scanners.json',
             'https://example.com/api/rest/v1/products/10661721',
         );
+        $this->mockPimAPIResponse(
+            'get-product-empty.json',
+            'https://example.com/api/rest/v1/products/empty',
+        );
```

Comment:
You also create a `get-products-empty.json` but do not add it to mock responses. Is it on purpose?

Reply:
Yes, because the API call for "GET /products" is already mocked by another prepared response.
You can only have one response for one url, so I override it in the beginning on my test:

https://github.com/akeneo/demo-app/pull/39/files#diff-4a891abee34c0176c4eab6812f204df6c24d72fd514be386f46ab17b9d38053fR65-R68